spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceConfiguration.java

```diff
@@ -83,6 +85,9 @@
 	@Autowired
 	private MessageBuilderFactory messageBuilderFactory;
 
+	@Lazy @Autowired(required = false)
```

Comment:
I just tried removing `@Lazy` on the ObjectMapper and `SpelExpressionConverterConfigurationTests` fails. I couldn't figure out what exactly causes this but I guess something is going on with the BeanPostProcessor.